<!-- launch rhino real time experiment --> 

<launch>
    <!-- args -->
    <arg name="exp_config" default="reduced_mu_turn_config.yaml"/>

    <!-- params -->
    <rosparam command="load" file="$(find common)/config/exp_config/$(arg exp_config)" />

    <!-- experiment manager node -->
    <node pkg="common" type="exp_manager.py" name="experiment_manager" output="screen"/>

    <!-- planning & ctrl nodes -->
    <rosparam command="load" file="$(find saarti)/config/planning_and_ctrl_config.yaml" /> 
    <node pkg="saarti" type="saarti_node" name="saarti_node" /> 
    <node pkg="saarti" type="ctrl_interface.py" name="ctrl_interface"/> 
    <node pkg="opendlv_ros" type="cmd_translator_node" name="cmd_translator"/> 

</launch>


